Atomic/syscontainers.py

```diff
@@ -669,6 +669,17 @@ def _canonicalize_location(self, destination):
             sysroot.load()
             osname = sysroot.get_booted_deployment().get_osname()
             destination = os.path.realpath(os.path.join("/ostree/deploy/", osname, os.path.relpath(destination, "/")))
```

Comment:
if we don't overwrite destination here, we can still use it instead of hardcoding "/var" (which is not always correct as the same function is used in user mode as well)

Reply:
Do you mean rename this instance of destination to something like ``ostree_destination``?